[CS2113-F11-3] TourPlanner

Bobowoo2468 opened the pull request:
TourPlanner is a desktop application meant for employees of travel agencies. Its main purpose is to manage clients, flights, accommodations and trip packages data, optimized for use via a Command Line Interface (CLI). If you can type fast, this application can allow one to access relevant travel information faster than traditional GUI applications.

Bobowoo2468 committed:
Merge pull request #81 from YipWayne/Wayne-v2.0b

Re-organised the classes, placed them in respective folders.

Bobowoo2468 committed:
Add sort functionality for Clients and Tours

For clients, sort by Id and name, in alphabetical order.

For tours, sort by TourCode and price, in ascending and alphabetical order respectively.

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/Ui.java
#	src/main/java/seedu/duke/commands/tours/SortTourCommand.java

Bobowoo2468 committed:
Edit to clear Java CI

Bobowoo2468 committed:
Merge pull request #82 from Bobowoo2468/master

Add sort functionality for Clients and Tours

swongts committed:
Add clientPackageId attribute to ClientPackage class, add and update cut class of Client, Tour, Flight and Package classes

swongts committed:
Fix gradle checks

Commit:
add's exception

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/CutClientCommand.java
#	src/main/java/seedu/duke/Parser.java
#	src/main/java/seedu/duke/SortTourCommand.java
#	src/main/java/seedu/duke/TourList.java
#	src/main/java/seedu/duke/Ui.java

Bobowoo2468 committed:
Merge pull request #91 from swongts/master

Add clientPackageId attribute to ClientPackage class, add and update cut class of Client, Tour, Flight and Package classes

Commit:
add's exception

Commit:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

 Conflicts:
	src/main/java/seedu/duke/Parser.java
	src/test/java/seedu/duke/ParserTest.java

Commit:
add's exception

Bobowoo2468 committed:
Update Ui.java

Bobowoo2468 committed:
Merge pull request #93 from Demonshaha/master

add's exception

Bobowoo2468 committed:
Add sort flights functionality

Bobowoo2468 committed:
Refresh Junit tests to pass all testing

Bobowoo2468 committed:
Clear checkstyle tests

swongts committed:
Add list of packages when using find client

Bobowoo2468 committed:
Merge pull request #94 from Bobowoo2468/master

Add sort flights functionality

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/Parser.java

irvinseet commented on docs/DeveloperGuide.md:
```diff
@@ -2,14 +2,73 @@
+
+After receiving the command from the user by `readCommand` of `Ui`, the command is parsed by `Parser`. The `Parser`
+first determines the command type by separating the command from the subsequent arguments. After determining the
+command, `Parser` executes the specific methods to sense-make the arguments, in response to the specific command.
```
You want want to add a diagram here showing the interaction between UI and Parser

Nirmaleshwar replied:
Maybe can also include an over-arching software architecture diagram to show how the high-level components connect.

irvinseet commented on docs/DeveloperGuide.md:
```diff
@@ -36,3 +102,77 @@
+
+The following activity diagram summarizes the following steps.
+
+![image](yuemel.png)
```
image not showing on github pages

AnShengLee replied:
Agree with @irvinseet, image is not showing up on github pages and github. On another note, would be good if a better name is given instead of `image`.

irvinseet commented on docs/DeveloperGuide.md:
````diff
@@ -36,3 +102,77 @@
+
+* Alternative: only iterate through the ```Package``` List.
+   * Pros: fast querying time.
+   * Cons: If the client has not subscribed to a package, none of their information can be accessed, including their contact number.
````
Formatting seems abit off here. Is this "Alternative" a standalone point. If yes, you may want to remove the bullet point for the first line

irvinseet commented on docs/DeveloperGuide.md:
```diff
@@ -36,3 +102,77 @@
+
+<br>
+
+### <u>Design Considerations</u>
```
does "design consideration" belong to "Finding a particular client". If so, please under different font sizes to indicate it is a subheading

irvinseet commented on docs/DeveloperGuide.md:
```diff
@@ -2,14 +2,73 @@
+* `add -c NAME /cn 91234567`
+* `add -t JPN /n Hokkaido-A /p 1500`
+* `add -f SQ-JPN /t JPN /f SG dt 20-10-2021 18:00 /df 21-10-2021 03:00`
+* `add -p ID /c NAME /t JPN /f SQ-JPN`
```
Perhaps can add the English equivalents of these commands

irvinseet commented on docs/DeveloperGuide.md:
````diff
@@ -2,14 +2,73 @@
+
+```
+-t JPN /n Hokkaido-A /p 1500 --> -t <<split>> JPN /n Hokkaido-A /p 1500 --> [-t, JPN /n Hokkaido-A /p 1500]
+```
````
For all the steps, you may want to separate and colour code the different attributes, then the splitting will be more visual and obvious to the reader. 

e.g.  /n Hokkaido-a (#red)    /p 1500 (#blue) -->   /n Hokkaido-a (#red)   <<SPLIT>>   /p 1500 (#blue)

irvinseet reviewed (commented):
No visuals used e.g., screenshots/diagrams. Although architecture diagram is optional, I think it is good to include as it gives readers and overview of your code.

irvinseet commented on docs/DeveloperGuide.md:
````diff
@@ -36,3 +102,77 @@
+* ```FindClientCommand(String name)```
+
+* ```FindTourCommand(String code)```
+
````
![image](https://user-images.githubusercontent.com/61399162/138996559-1b00966d-6af5-4211-8216-6996bd7a24f2.png)

Inconsistent format for classes

Commit:
storage

Commit:
Merge remote-tracking branch 'origin/master'

YipWayne committed:
Merge branch 'master' into master

Bobowoo2468 committed:
Merge pull request #98 from swongts/master

Add list of packages when using find client

Bobowoo2468 committed:
Update Parser.java

Bobowoo2468 committed:
Update Parser.java

Bobowoo2468 committed:
Merge pull request #99 from Demonshaha/master

Storage

Bobowoo2468 committed:
Delete TourList

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Fix storage to run Main

swongts committed:
Merge pull request #101 from Bobowoo2468/master

Fix storage to run program

Bobowoo2468 committed:
Complete storage functionality

Bobowoo2468 committed:
Merge pull request #102 from Bobowoo2468/master

Complete storage functionality

yipwayyne committed:
Committed Find JUnit Test, minor bug in ParseFind

YipWayne committed:
Merge branch 'master' into Wayne-v2.0b

YipWayne committed:
Update FindClientPackageCommandTest.java

YipWayne committed:
Update FindFlightCommandTest.java

YipWayne committed:
Update FindTourCommandTest.java

Bobowoo2468 committed:
Merge pull request #103 from YipWayne/Wayne-v2.0b

Committed Find JUnit Test, minor bug in ParseFind

Bobowoo2468 committed:
Add parse exception for duplicate functions and refactored parser code per TA's feedback

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/Parser.java

AnShengLee commented on docs/DeveloperGuide.md:
```diff
@@ -2,14 +2,73 @@
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
```
Might want to remove this prompt for final version.

AnShengLee commented on docs/DeveloperGuide.md:
```diff
@@ -36,3 +102,77 @@
+Depending on the type of list command being called, these command types will be returned:
+* `list -c`: `ListCCommand`
+* `list -t`: `ListTCommand`
+* `list -p`: `ListPCommand`
```
Would be nice if there is a sequence diagram to illustrate how `list -c` works.

Nirmaleshwar commented on docs/DeveloperGuide.md:
````diff
@@ -36,3 +102,77 @@
+
+**Step 3**: The ```FindClientCommand``` iterates through each ```Client``` object in the ```ClientList```. For every ```Client```, the ```getName()``` function is called to retrieve the name attribute of the Client. If the name attribute is equals to ```Bo Tuan```, the ```Client``` object is printed onto the console terminal.
+
+</br>
````
This line break appears on the webpage. You can remove the forward-slash from the line break and that should fix the issue.

AnShengLee commented on docs/DeveloperGuide.md:
```diff
@@ -36,3 +102,77 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
```
Prompt to be removed for final version.

Nirmaleshwar commented on docs/DeveloperGuide.md:
```diff
@@ -36,3 +102,77 @@
+
+**Step 2**: `parse()` identifies the command type `ListCCommand` based on the user input and returns it.
+
+**Step 3**: Then, `execute()` method in `ListCCommand` is called, where it loops through the current `clientList` and prints out all client names and their details.
```
Perhaps you can illustrate these steps by showing the flow of the method using a suitable UML diagram. That might help give a reader a better overview.

AnShengLee commented on docs/DeveloperGuide.md:
```diff
@@ -20,10 +79,17 @@
+|`* * *`|user|add a new tour package|update the tour package database|
+|`* * *`|user|delete a tour package|update the tour package database|
+|`* * *`|user|add flight timings|keep track of flights|
+
```
Nice user stories, there is even a column to show priority. Maybe can move this section below. More focus should be put on the implementation of different components.

Nirmaleshwar commented on docs/DeveloperGuide.md:
```diff
@@ -20,10 +79,17 @@
 |--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+|`* * *`|new user|see usage instructions|refer to them when I forget how to use the application|
```
Maybe in addition to the priority, can also include the version in which the feature will be included by.

AnShengLee reviewed (commented):
Overall, there are some missing information in the DG such as important class diagrams, sequence diagrams and descriptions (i.e., User profile and Value proposition). The descriptions of functions are there. They just need diagram to help better illustrate and explain the points.

yipwayyne committed:
Comitted JUnit testing for List Commands

YipWayne committed:
Merge pull request #104 from Bobowoo2468/master

Add parse exception for duplicate functions

YipWayne committed:
Update ListClientCommandTest.java

YipWayne committed:
Update ListClientPackageCommandTest.java

YipWayne committed:
Update ListClientPackageCommandTest.java

YipWayne committed:
Update ListFlightCommandTest.java

YipWayne committed:
Update ListClientPackageCommandTest.java

YipWayne committed:
Update ListTourCommandTest.java

YipWayne committed:
Update FindFlightCommandTest.java

YipWayne committed:
Update ListClientPackageCommandTest.java

yipwayyne committed:
Committ v2.0 of UG, minor bug fixes

yipwayyne committed:
Committed back original ver of TourPlanner.txt

yipwayyne committed:
Committed * error for some classes

Bobowoo2468 committed:
Merge pull request #105 from YipWayne/Wayne-v2.0

Comitted JUnit testing for List Commands

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.0

swongts committed:
Add cutClientCommand junit test

swongts committed:
Merge pull request #106 from YipWayne/Wayne-v2.0

UG v2.0 find and list

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	data/TourPlanner.txt

Bobowoo2468 committed:
TourPlanner v2.0 Release

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	data/TourPlanner.txt

Bobowoo2468 committed:
Merge pull request #109 from Bobowoo2468/master

TourPlanner V2.0 Release

Bobowoo2468 committed:
Merge pull request #107 from swongts/master

Add cutClientCommand junit test

Bobowoo2468 committed:
Incremental add for UG add command

yipwayyne committed:
Minor edits for find function in UG, fleshed out find in DG. Bug fixes for find client

YipWayne committed:
Merge branch 'master' into Wayne-v2.0

Bobowoo2468 committed:
Merge pull request #110 from Bobowoo2468/master

Incremental add for UG add command

Bobowoo2468 committed:
Merge pull request #111 from YipWayne/Wayne-v2.0

Minor edits for find function in UG, fleshed out find in DG. Bug fixe…

swongts committed:
Add cut command to user guide, show client package for cut client/flight/tour

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Merge pull request #112 from swongts/master

Add cut command to user guide, show client package for cut client/flight/tour

swongts committed:
Update add client package, must have unique client package id

Bobowoo2468 committed:
Finish up add command documentation for User Guide

YipWayne committed:
Merge pull request #113 from swongts/master

Update add client package, must have unique client package id

YipWayne committed:
Merge pull request #114 from Bobowoo2468/master

Finish up add command documentation for User Guide

Bobowoo2468 committed:
Add exceptions for missing sort identifiers for Sort Command

YipWayne committed:
Merge pull request #115 from Bobowoo2468/master

Add exceptions for missing sort identifiers for Sort Command

Demonshaha committed:
Update UserGuide.md

Demonshaha committed:
Merge branch 'AY2122S1-CS2113T-F11-3:master' into master

Demonshaha committed:
Update UserGuide.md

Bobowoo2468 committed:
Merge pull request #116 from Demonshaha/master

Update UserGuide.md

Bobowoo2468 committed:
Update UserGuide.md

Bobowoo2468 committed:
Update User Guide to fix documentation bugs and Storage

yipwayyne committed:
Committed Start and end content of DG,

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.0

# Conflicts:
#	src/main/java/seedu/duke/commands/clientpackages/AddClientPackageCommand.java

Bobowoo2468 committed:
Merge pull request #118 from YipWayne/Wayne-v2.0

DG front and back content

Bobowoo2468 committed:
Add Junit testing for all Sort Commands

Bobowoo2468 committed:
Edit to clear Checkstyle errors

Bobowoo2468 committed:
Add changes to parser test to include testing for duplicate prefixes

swongts committed:
Add jUnit testing for cutFlightCommand, cutTourCommand

swongts committed:
Add help command

Bobowoo2468 committed:
Merge pull request #121 from swongts/master

Add jUnit testing for cutFlightCommand, cutTourCommand

yipwayyne committed:
Fix UG-related issues

yipwayyne committed:
Organised points in UG, added emojis

YipWayne committed:
Merge pull request #162 from YipWayne/Wayne-v2.1

Fix UG-related issues

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/data/TourList.java
#	src/test/java/seedu/duke/AddFlightCommandTest.java
#	src/test/java/seedu/duke/commands/clients/CutClientCommandTest.java

Bobowoo2468 committed:
Merge pull request #119 from Bobowoo2468/master

Add Junit testing for Sort Commands

swongts committed:
Add addClientPackageCommandTest

swongts committed:
Fix gradle checks

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/test/java/seedu/duke/commands/clients/CutClientCommandTest.java

Bobowoo2468 committed:
Merge pull request #163 from swongts/master

Add addClientPackageTest

Bobowoo2468 committed:
Add flight and client exception for erroneous email and contact number, and wrong date time formatting and entry

Bobowoo2468 committed:
no message

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/Ui.java

yipwayyne committed:
Committed UG (should be mostly final save for missing info)

Bobowoo2468 committed:
Merge pull request #164 from Bobowoo2468/master

Add flight and client exception

Bobowoo2468 committed:
Add Storage functionality (some implementation by ChengXu) and further parse exceptions

swongts committed:
Add Command and cutCommand to developer's guide

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

swongts committed:
Fix DG diagram bug

YipWayne committed:
Merge pull request #167 from swongts/master

Add Command and cutCommand to developer's guide

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

swongts committed:
Merge pull request #166 from Bobowoo2468/master

Add Storage functionality (some implementation by ChengXu)

swongts committed:
Merge pull request #165 from YipWayne/Wayne-v2.1

Committed UG (should be mostly final save for missing info)

yipwayyne committed:
Added Ui component in UG, minor changes to UML diagrams

Bobowoo2468 committed:
Merge pull request #172 from YipWayne/Wayne-v2.1

Added Ui component to DG, minor fixes to UML diagrams

YipWayne committed:
Added images yayerz

Bobowoo2468 committed:
Add TOC to UG and How-To-Use UG

Add bugfix to include sorting with case insensitivity. Add sort tour by name functionality.

Bobowoo2468 committed:
Update README.md

Bobowoo2468 committed:
Update README.md

Bobowoo2468 committed:
Merge pull request #174 from Bobowoo2468/master

Add TOC to UG and How-To-Use UG

swongts committed:
Add sortCommand to DG

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	docs/DeveloperGuide.md

swongts committed:
Fix clientPackageStorage bug

swongts committed:
Delete unnecessary comments

Bobowoo2468 committed:
Merge pull request #175 from swongts/master

Add sortCommand to DG, fix storageClientPackage bug

yipwayyne committed:
Committed TOC for DG

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

Bobowoo2468 committed:
Add sequence diagram to Parser, and made changes to pass Junit testing

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	docs/DeveloperGuide.md

Bobowoo2468 committed:
Add bugfix to method in lists, and JavaDoc

Bobowoo2468 committed:
Merge pull request #176 from Bobowoo2468/master

Add Parser to DG

swongts committed:
Add javadoc for cut related commands and client package

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/Parser.java
#	src/main/java/seedu/duke/data/ClientList.java
#	src/main/java/seedu/duke/storage/ClientPackageStorage.java

swongts committed:
add jUnit test for CutClientPackageCommand, HelpCommand and ByeCommand

Demonshaha committed:
Update DeveloperGuide.md

swongts committed:
Fix DG UML diagram bugs, refactor magic Strings in test for cut commands

swongts committed:
Merge pull request #177 from swongts/master

Add javadoc, add junit testing, fix UML bugs in DG

Bobowoo2468 committed:
Add addCommand and Parser to DG

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	docs/DeveloperGuide.md

Bobowoo2468 committed:
Clear Checkstyle tests

Bobowoo2468 committed:
Update DeveloperGuide.md

Bobowoo2468 committed:
Add PPP, and add tests to Parser

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

swongts committed:
Merge pull request #181 from Bobowoo2468/master

Add addCommand and Parser to DG

YipWayne committed:
Merge branch 'master' into Wayne-v2.1

yipwayyne committed:
Changed ListCommandTest and FindCommandTest, Edited DG and UML diagrams

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

# Conflicts:
#	docs/DeveloperGuide.md
#	src/main/java/seedu/duke/Parser.java
#	src/main/java/seedu/duke/commands/clients/AddClientCommand.java
#	src/main/java/seedu/duke/data/FlightList.java
#	src/main/java/seedu/duke/data/TourList.java

swongts committed:
add PPP, add javadoc for Storage and TourPlanner, refactor TourPlanner, update user guide

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/TourPlanner.java

Bobowoo2468 committed:
Merge pull request #189 from YipWayne/Wayne-v2.1

 Changed ListCommandTest and FindCommandTest, Edited DG and UML diagrams

Bobowoo2468 committed:
Update Storage Class diagram (Chengxu)

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Set theme jekyll-theme-time-machine

Bobowoo2468 committed:
Update _config.yml

To add emojis to Jekyll-hosted pages

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

yipwayyne committed:
Committed YipWayne's PPP

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

# Conflicts:
#	docs/DeveloperGuide.md

YipWayne committed:
Merge pull request #192 from YipWayne/Wayne-v2.1

YipWayne section in PPP

YipWayne committed:
Merge pull request #190 from swongts/master

add PPP, add javadoc for Storage and TourPlanner, refactor TourPlanner, update user guide

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

YipWayne committed:
Update DeveloperGuide.md

swongts committed:
Change AddClientPackageCommand

Demonshaha committed:
Create Demonshaha.md

Demonshaha committed:
Update Demonshaha.md

Demonshaha committed:
Update Demonshaha.md

Bobowoo2468 committed:
Add UML diagram for DG for add client package

Demonshaha committed:
add author declare

Demonshaha committed:
Update ClientStorage.java

Demonshaha committed:
Update FlightStorage.java

Demonshaha committed:
Update TourStorage.java

swongts committed:
FIx gradle checks

swongts committed:
Fix gradle checks 2

swongts committed:
Merge branch 'master' into master

swongts committed:
Fix gradle checks 3

swongts committed:
Edit DG

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	src/main/java/seedu/duke/commands/clients/AddClientCommand.java
#	src/main/java/seedu/duke/commands/tours/AddTourCommand.java
#	src/main/java/seedu/duke/storage/ClientStorage.java
#	src/main/java/seedu/duke/storage/FlightStorage.java
#	src/main/java/seedu/duke/storage/TourStorage.java

swongts committed:
Merge branch 'master' of https://github.com/swongts/tp

swongts committed:
Merge pull request #193 from swongts/master

Change AddClientPackageCommand

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Merge pull request #194 from Bobowoo2468/master

Edit DG and sequence diagrams to add command

swongts committed:
Add UML parser diagram

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Add JavaDoc and rearranged code

Bobowoo2468 committed:
Deleted storage test

Bobowoo2468 committed:
Merge pull request #195 from Bobowoo2468/master

Add JavaDoc and rearranged code

Bobowoo2468 committed:
Add logger functionality

Demonshaha committed:
Update ClientPackageStorage.java

swongts committed:
Merge pull request #196 from Bobowoo2468/master

Add logger functionality

Demonshaha committed:
Update ClientStorage.java

Demonshaha committed:
Update FlightStorage.java

Demonshaha committed:
Update TourStorage.java

swongts committed:
Clean up DG

Bobowoo2468 committed:
Add image to AboutUs

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

swongts committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

yipwayyne committed:
Commit Javadoc for some Ui functions, change to findclient UML

Demonshaha committed:
Update TourStorage.java

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

# Conflicts:
#	docs/findclient.PNG
#	src/main/java/seedu/duke/Ui.java

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

swongts committed:
Fix gradle checks

Bobowoo2468 committed:
Merge pull request #197 from swongts/master

clean up DG

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Merge pull request #198 from Bobowoo2468/master

Edit AboutUs to link PPP

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

YipWayne committed:
wen face

YipWayne committed:
Update AboutUs.md

YipWayne committed:
Updated picture for Wayne 

help

Bobowoo2468 committed:
Changed images for DG and Logger for TourPlanner

Bobowoo2468 committed:
Update bobowoo2468.md

Bobowoo2468 committed:
Update bobowoo2468.md

Bobowoo2468 committed:
Update bobowoo2468.md

Bobowoo2468 committed:
Update UG

Bobowoo2468 committed:
Add picture to AboutUs

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

# Conflicts:
#	docs/AboutUs.md

Bobowoo2468 committed:
Merge pull request #199 from Bobowoo2468/master

Changed images for DG and Logger for TourPlanner

Bobowoo2468 committed:
Update bobowoo2468.md

swongts committed:
Add appendix to DG

swongts committed:
Merge pull request #200 from swongts/master

Add appendix to DG

Bobowoo2468 committed:
Add minor changes to DG

Bobowoo2468 committed:
Merge pull request #201 from Bobowoo2468/master

Add minor changes to DG

YipWayne committed:
Update Ui.java

Bobowoo2468 committed:
Merge pull request #202 from YipWayne/Wayne-v2.1

Javadoc comments for Ui

YipWayne committed:
Update DeveloperGuide.md

Bobowoo2468 committed:
Add major bugfix to addCommand

yipwayyne committed:
Made new title lermaooooo

yipwayyne committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp into Wayne-v2.1

# Conflicts:
#	src/main/java/seedu/duke/Ui.java

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

YipWayne committed:
Update YipWayne.md

Bobowoo2468 committed:
Add findClient sequence diagram to DG

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Merge pull request #204 from YipWayne/Wayne-v2.1

Gigabrain new logo

YipWayne committed:
Merge pull request #203 from Bobowoo2468/master

Add major bugfix to addCommand

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

yipwayyne committed:
FindClient bug fix

Bobowoo2468 committed:
Merge pull request #205 from YipWayne/Wayne-v2.1

FindClient bug fix

swongts committed:
Fix null pointer bug

Bobowoo2468 committed:
Merge pull request #206 from swongts/master

Fix null pointer bug

YipWayne committed:
Update YipWayne.md

swongts committed:
Fix clientPackageCommand bug

Bobowoo2468 committed:
Delete PNG file

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Merge pull request #207 from swongts/master

Fix clientPackageCommand bug

Bobowoo2468 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F11-3/tp

Bobowoo2468 committed:
Add changes for bugfix

Bobowoo2468 committed:
Change to clear Java CI tests

Bobowoo2468 committed:
Merge pull request #208 from Bobowoo2468/master

Add bugfix to code

Bobowoo2468 committed:
Update UserGuide.md

Bobowoo2468 committed:
Update UserGuide.md
